packages/libprotobuf/build.sh

```diff
@@ -8,9 +8,9 @@ TERMUX_PKG_MAINTAINER="@termux"
 # - update SHA256 checksum for $_PROTOBUF_ZIP in
 #     $TERMUX_SCRIPTDIR/scripts/build/setup/termux_setup_protobuf.sh
 # - ALWAYS bump revision of reverse dependencies and rebuild them.
-TERMUX_PKG_VERSION=2:22.4
+TERMUX_PKG_VERSION=4:24.3
```

Comment:
Why are you bumping the package version epoch from `2:` to `4:`?
This does not appear to be done by [any other package of Protobuf](https://repology.org/project/protobuf/packages).

Reply:
The epoch for `libprotobuf` was previous incremented from
`1:` to `2:` when the source repository was moved
See: https://github.com/termux/termux-packages/commit/16d55adac2a7e583581b7a407edaf2259b19d8dd

And previously from none to `1:` when a downgrade was required due to issues.
See: https://github.com/termux/termux-packages/commit/a6910acef563063cd8dce0735ec13a5a5836f5e3

This does not appear to be a situation warranting an epoch bump, and if it were it would make more sense to bump it to `3:`

Reply:
No need to bump epoch. Keep it at 2.
24.3 is still greater than 22.4 when comparing version.